### PR TITLE
Fixes #24399 - Drop email configuration via files

### PR DESCRIPTION
**Pull request by ekohl**
This has been deprecated since 1.14 and removed in 1.16.

### CODE REVIEW

**Comment by sean797** [comment]
Do we need or want an installer migration to remove those parameters?

**Comment by ekohl** [comment]
@sean797 probably needed yes.

File: manifests/config.pp [hunk 1]
```diff
@@ -25,20 +25,9 @@
...
   }
 
-  if $::foreman::email_delivery_method and !empty($::foreman::email_delivery_method) {
-    if $::foreman::email_config_method == 'file' {
-      file { "/etc/foreman/${foreman::email_conf}":
-        ensure  => file,
-        owner   => 'root',
-        group   => $::foreman::group,
-        mode    => '0640',
-        content => template("foreman/${foreman::email_source}"),
```
Comment (tbrisker) [inline manifests/config.pp]: You can also remove the template itself I believe.

Comment (ekohl) [reply manifests/config.pp]: Done

**Comment by ekohl** [comment]
Acceptance tests are failing because of https://github.com/puppetlabs/puppetlabs-postgresql/pull/1012

**Comment by mmoll** [comment]
merged, bedankt @ekohl!

**Pull request merged**